sequencer/src/api/update.rs

```diff
@@ -47,3 +58,68 @@ where
 
     Ok(())
 }
+
+#[async_trait]
+impl<N, D, Ver: StaticVersionType> UpdateStateStorage<SeqTypes, StorageState<N, D, Ver>>
```

Comment:
This is defined twice?

Reply:
Oh yeah this should be a wrapper impl which uses the inner data source